src/Controls/src/Core/Element/Element.cs

```diff
@@ -56,14 +58,17 @@ public string AutomationId
 			}
 		}
 
-		/// <include file="../../docs/Microsoft.Maui.Controls/Element.xml" path="//Member[@MemberName='ClassId']/Docs/*" />
+		/// <summary>Gets or sets a value used to identify a collection of semantically similar elements.</summary>
+		/// <value>A string that represents the collection the element belongs to.</value>
+		/// <remarks>Use the class id property to collect together elements into semantically similar groups for identification in ui testing and in theme engines.</remarks>
 		public string ClassId
 		{
 			get => (string)GetValue(ClassIdProperty);
 			set => SetValue(ClassIdProperty, value);
 		}
 
-		/// <include file="../../docs/Microsoft.Maui.Controls/Element.xml" path="//Member[@MemberName='Effects']/Docs/*" />
+		/// <summary> Gets or sets the styles and properties that will be applied to the element during runtime.</summary>
+		/// <value> A collection containing the different <see cref="Effect"/> to be applied to the element</value>
```

Comment:
```suggestion
		/// <summary>Gets or sets the styles and properties that will be applied to the element during runtime.</summary>
		/// <value>A collection containing the different <see cref="Effect"/> to be applied to the element</value>
```

Reply:
Add a period to the end of the sentence for the value description.